subworkflows/local/bam_variant_calling_somatic_all/main.nf

```diff
@@ -175,11 +174,10 @@ workflow BAM_VARIANT_CALLING_SOMATIC_ALL {
     }
 
     // MSISENSOR
-    if (tools.split(',').contains('msisensorpro')) {
+    if (tools.split(',').contains('msisensorpro') && msisensorpro_scan) {
```

Comment:
This should not be strictly necessary since the empty channel should skip the tool

Reply:
I remember I tried without it but it failed for some reason that I can't remember now so I think it was necessary.